common/app/layout/Front.scala

```diff
@@ -365,10 +367,15 @@ object Front extends implicits.Collections {
     )
   }
 
+  private def applyShowMoreExperiment(faciaContainer: FaciaContainer)(implicit request: RequestHeader): FaciaContainer = {
+    val showMoreEnabled = !ActiveExperiments.isParticipating(HideShowMoreButtonExperiment)
+    faciaContainer.copy(hasShowMoreEnabled = showMoreEnabled)
+  }
+
   def fromPressedPage(pressedPage: PressedPage,
```

Comment:
unrelated to this PR but those `fromPressedPage...` function should be move to Facia which is the only placed they are called from

Reply:
Nice spot. I'll do this refactor in a separate PR